raiden/raiden_service.py

```diff
@@ -324,9 +328,10 @@ def start(self):
                 [],  # empty list of token network states as it's the node's startup
             )
             state_change = ContractReceiveNewPaymentNetwork(
-                constants.EMPTY_HASH,
-                payment_network,
-                last_log_block_number,
+                transaction_hash=constants.EMPTY_HASH,
```

Comment:
@pirapira could you please add the transaction hash for contract deploy in the json files?

Reply:
Will be done https://github.com/raiden-network/raiden-contracts/issues/582

Reply:
@hackaugusto I already see `transaction_hash` fields.  What is missing?

Reply:
@LefterisJP can you please use the available data? (I assume it's also exposed through the contract manager)